src/net/native-stack-impl.hh

```diff
@@ -244,7 +245,8 @@ native_connected_socket_impl<Protocol>::get_nodelay() const {
 template <typename Protocol>
 void native_connected_socket_impl<Protocol>::set_keepalive(bool keepalive) {
     // FIXME: implement
-    std::cerr << "Keepalive is not supported by native stack" << std::endl;
+    std::fprintf(stderr, "Keepalive is not supported by native stack\n");
+    std::fflush(stderr);
```

Comment:
You can use seastar_logger for these things.

Reply:
yeah, good idea. applied.